fb2k/in_vgmstream.cpp

```diff
@@ -250,6 +252,11 @@ void input_vgmstream::decode_seek(double p_seconds,abort_callback & p_abort) {
 	int max_buffer_samples = sizeof(sample_buffer)/sizeof(sample_buffer[0])/vgmstream->channels;
 	bool loop_okay = loop_forever && vgmstream->loop_flag && !ignore_loop && !force_ignore_loop;
 
+	/* start on first channel to avoid problems in some decoders */
```

Comment:
I was under the impression seek_pos_samples=0 would mean first sample in first channel, seek_pos_samples=1 first sample in second channel, and so on.
I guess I misunderstand the issue. Here is one easy test from [FF XI](https://github.com/kode54/vgmstream/files/706992/music076.zip). Seek and half of the time you'll get garbled sounds (hence I thought it was landing on a sample of an odd channel).

This happened before any of my IO changes.